pkg/JuliaInterface/init.g

```diff
@@ -4,22 +4,46 @@
 # Reading the declaration part of the package.
 #
 
-##
-##  If the value is <K>true</K> then the file
-##  <F>~/.julia/config/startup.jl</F>
-##  gets imported into &Julia; after startup.
-##  If the value is a nonempty string that is the name of a directory
-##  then the <F>startup.jl</F> file in this directory gets imported into
-##  &Julia;.
-##  Otherwise no <F>startup.jl</F> file will be imported automatically,
-##  since apparently <C>libjulia</C> does not import such a file.
+#! @BeginChunk IncludeJuliaStartupFile
+#! @Chapter Introduction to <Package>JuliaInterface</Package>
+#! @Section User preferences in the <Package>JuliaInterface</Package> package
```

Comment:
Try removing these lines to see if it helps with the chunk issue.

If not, let's merge this PR anyway, we can figure this out later (perhaps log an issue so we don't forget)
```suggestion
```

Reply:
Yes, this helps.
(Strange, I remember that I got error messages because of the missing "context" in similar situations.)